adagrams/game.py

```diff
@@ -1,13 +1,101 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+SCORE_CHART = { "A":1, "E":1, "I":1, "O":1, "U":1, "L":1, "N":1, "R":1, "S":1, "T":1, 
+            "D":2,"G":2, 
+            "B":3, "C":3, "M":3, "P":3,
+            "F":4, "H":4, "V":4, "W":4, "Y":4,
+            "K":5,
+            "J":8,"X":8,
+            "Q":10,"Z":10
+            }
+
 def draw_letters():
-    pass
+    #this is the array that will be returned
+    list_ten_strings = []
 
-def uses_available_letters(word, letter_bank):
+    #new list called bag to add the random keys frequency
+    bag = []
+        
+    for a_single_letter, count in LETTER_POOL.items():
+        for _ in range(count):
+            bag.append(a_single_letter)
+
+    while len(list_ten_strings) < 10:
```

Comment:
In Python, 10 on this line is what is known as a "Magic Number". We use the term to refer to hard coded values that just kind of appear without any context to them!

If I did not know how to play Adagrams, I would not understand the significance of 10 here. If I were a dev who tried to read this, I might be wondering "Why do we only loop 10 times"?

It would be more descriptive to use a constant variable to reference 10 so that your code is more self-documenting.

```suggestion
    NUM_TILES_ALLOWED_IN_HAND = 10
    while len(list_ten_strings) < NUM_TILES_ALLOWED_IN_HAND:
```